### ISSUE TITLE
Support for Visual Studio 2019 preview 1?

### ISSUE THREAD

**Issue by IanKemp** [JosefPihrt/Roslynator#468]
https://blogs.msdn.microsoft.com/visualstudio/2018/12/04/making-every-developer-more-productive-with-visual-studio-2019/

I tried installing Roslynator 17, both via the vsix and via VS2019's "Extensions and Updates" dialog, and it didn't show up. Roslynator has become an extension I literally cannot work without, so I basically can't use VS2019 until this extension is available there. 

 I appreciate that having to support an extension for an alpha product is unrealistic, so if you could just post instructions on how I can build the extension from source that would be fine.

I had a look at [source.extension.vsixmanifest](https://github.com/JosefPihrt/Roslynator/blob/master/src/VisualStudio/source.extension.vsixmanifest) which I assume is the file responsible for allowing extensions to be installed in the various VS versions, but it seems you already have 16.0/VS2019 in there, so not sure why it isn't working.

**Comment by IanKemp** [comment]
This might be useful: https://blogs.msdn.microsoft.com/visualstudio/2018/09/26/how-to-upgrade-extensions-to-support-visual-studio-2019/

tl;dr update 16.0 to 17.0 (which doesn't make sense since the version of VS2019 is 16.0 according to Help/About, but whatever Microsoft), except for the `<Prerequisite>` versions which should be open-ended.

**Comment by IanKemp** [comment]
Just forked the repo and made the changes suggested by Mads' blog post (including removal of the `Microsoft.VisualStudio.MPF` dependency) to `source.extension.vsixmanifest`, then did a successful build and opened the resulting `.vsix`. The VSIX installer allowed me to install the extension into VS2019 and after restarting VS2019, Roslynator works exactly as it did in VS2017.

PR linked.

**Comment by Daddoon** [comment]
@IanKemp Work like a charm ! Thanks !

**Comment by vip32** [comment]
> Just forked the repo and made the changes suggested by Mads' blog post (including removal of the `Microsoft.VisualStudio.MPF` dependency) to `source.extension.vsixmanifest`, then did a successful build and opened the resulting `.vsix`. The VSIX installer allowed me to install the extension into VS2019 and after restarting VS2019, Roslynator works exactly as it did in VS2017.
> 
> PR linked.

PR?

**Comment by JosefPihrt** [comment]
@IanKemp  is working on the PR.

**Comment by CoolOppo** [comment]
@JosefPihrt nice, glad to hear

@vip32 that is the funniest profile pic I've seen in a while, excellent work

**Comment by JosefPihrt** [comment]
https://marketplace.visualstudio.com/items?itemName=josefpihrt.Roslynator2019
